### ISSUE TITLE
frozen BatchNorm

### ISSUE THREAD

**Issue by dereyly** [open-mmlab/mmdetection#113]
mmdetection use common BatchNorm without freeze? 
In SSD series batchNorm is unfrozen
I detectron and other faster RCNN batchNorm is frozen.

I want to train with BN and big batch size 4*16. Do I need change something?

PS: this trick is used in SNIPER and in competion WIDER winner.

**Comment by hellock** [comment]
You can set the arguments `bn_eval` and `bn_frozen` of `backbone` field in the config file to `False`, then parameters of BN layers will be updated during training.

**Comment by dereyly** [comment]
@hellock Thank you!
Batch Norm train with "synchronize BN"?

**Comment by hellock** [comment]
Synchronized BN is currently not supported and we have the plan to add it later. If you set `bn_eval` and `bn_frozen` as False, the normal BN is used.
